security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,153 +12,32 @@ These release notes track the development of {cert-manager-operator}.
-
-[id="cert-manager-operator-release-notes-1.11.1"]
-== Release notes for {cert-manager-operator} 1.11.1
-
-Issued: 2023-06-21
-
-The following advisory is available for the {cert-manager-operator} 1.11.1:
-
-* link:https://access.redhat.com/errata/RHEA-2023:3439[RHEA-2023:3439]
-
-Version `1.11.1` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.11.1`. For more information, see the link:https://cert-manager.io/docs/release-notes/release-notes-1.11/#v1111[cert-manager project release notes for v1.11.1].
-
-[id="cert-manager-operator-1.11.1-new-features-and-enhancements"]
+[id="cert-manager-operator-new-features-1.13"]
 === New features and enhancements
 
-This is the general availability (GA) release of the {cert-manager-operator}.
-
-[id="cert-manager-log-level-1.11.1"]
-==== Setting log levels for cert-manager and the {cert-manager-operator}
-* To troubleshoot issues with cert-manager and the {cert-manager-operator}, you can now configure the log level verbosity by setting a log level for cert-manager and the {cert-manager-operator}. For more information, see xref:../../security/cert_manager_operator/cert-manager-log-levels.adoc#cert-manager-log-levels[Configuring log levels for cert-manager and the {cert-manager-operator}].
-
-[id="cert-manager-authentication-aws-1.11.1"]
-==== Authenticating the {cert-manager-operator} with AWS
-* You can now configure cloud credentials for the {cert-manager-operator} on AWS clusters with Security Token Service (STS) and without STS. For more information, see xref:../../security/cert_manager_operator/cert-manager-authenticate-aws.adoc#cert-manager-authenticate-aws[Authenticating the {cert-manager-operator} on AWS  Security Token Service] and xref:../../security/cert_manager_operator/cert-manager-authentication-non-sts.adoc#cert-manager-authentication-non-sts[Authenticating the {cert-manager-operator} on AWS].
-
-[id="cert-manager-authentication-gcp-1.11.1"]
-==== Authenticating the {cert-manager-operator} with GCP
-* You can now configure cloud credentials for the {cert-manager-operator} on GCP clusters with Workload Identity and without Workload Identity. For more information, see xref:../../security/cert_manager_operator/cert-manager-authenticate-gcp.adoc#cert-manager-authenticate-gcp[Authenticating the {cert-manager-operator} with GCP Workload Identity] and xref:../../security/cert_manager_operator/cert-manager-authenticate-non-sts-gcp.adoc#cert-manager-authenticate-non-sts-gcp[Authenticating the {cert-manager-operator} with GCP]
+* You can now manage certificates for API Server and Ingress Controller by using the {cert-manager-operator}. 
```

Comment:
```suggestion
* You can now manage certificates for API Server and `IngressController` object by using the {cert-manager-operator}. 
```

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#ingress

Reply:
I am using this as a noun as per this
https://redhat-documentation.github.io/supplementary-style-guide/#ingress-controller